app/routes/account.js

```diff
@@ -16,7 +16,7 @@ export default TravisRoute.extend({
   },
 
   afterModel(model) {
-    if (model && !model.error)
+    if (model && !model.error && !this.get('features.enterpriseVersion'))
```

Comment:
I would suggest to explicitly inject `features` service on this route. Even if it works now due to implicit injection, it may change eventually (Ember team tends to get rid of all implicit "magic" over time) and the error will appear again.